### DISCUSSION TITLE
Is this a follow up to systemexplorer.net?

### DISCUSSION THREAD

**Discussion by kinngrimm** [zodiacon/SystemExplorer#13] [Q&A]
Is this the same tool or just has a similar/same name?

**Comment by zodiacon** [comment]
I am not familiar with that tool :)
Pure coincidence.

**Comment by blackcrack** [comment]
Do you meant THE System Explorer @kinngrimm ?  https://systemexplorer.net/
have still the un-bugged portable version ;) 7.0.0 and not the 7.0.1 version where was bug'ed by the headdeveloper.. ;) there was  troubles between the Developers.. i guess this "System Explorer" is an other Program.. fresh programmed ..

[https://www.blackysgate.de/files/folder-root/System%20Explorer/](https://www.blackysgate.de/files/folder-root/System%20Explorer/)

best
Blacky
